dashboard/db/migrate/20160505193334_create_peer_reviews.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePeerReviews < ActiveRecord::Migration
+  def change
+    create_table :peer_reviews do |t|
+      t.references :user, index: true, foreign_key: true
+      t.boolean :from_instructor, null: false, default: false
+      t.references :script, index: true, foreign_key: true, null: false
+      t.references :level, index: true, foreign_key: true, null: false
+      t.references :level_source, index: true, foreign_key: true, null: false
```

Comment:
The FK annotation should only be needed if it's something other than the default that Rails expects.